Competition/src/main/java/competition/subsystems/gripper/commands/ToggleGrabDiscCommand.java

```diff
@@ -0,0 +1,46 @@
+package competition.subsystems.gripper.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.gripper.GripperSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class ToggleGrabDiscCommand extends BaseCommand {
+    
+    final GripperSubsystem gripperSubsystem;
+
+    private String state; 
+
+    @Inject
+    public ToggleGrabDiscCommand(GripperSubsystem gripper, GrabDiscCommand grabDisc, ReleaseDiscCommand releaseDisc) {
+        gripperSubsystem = gripper;
+        state = "grab";
```

Comment:
All of this state stuff should be moved into the GripperSubsystem. It's too easy to lose state if, for example, your command is interrupted.

I suggest writing a `toggleGrab()` method in the GripperSubsystem, and changing the state string into an Enum. That way, this command would work no matter the initial state of the system.

Reply:
I moved the method to GripperSubsystem